Sources/PluginViewController/PluginViewController.swift

```diff
@@ -9,25 +9,37 @@ final class PluginViewController: UIViewController {
 
     var closeEventHandler: (() -> Void)?
 
-    private(set) lazy var tableView: UITableView = {
-        let tableView = UITableView(frame: .zero, style: .grouped)
-        tableView.delegate = configuration
-        tableView.dataSource = configuration
-        tableView.estimatedRowHeight = 100
-        tableView.separatorInset = .init(top: 0, left: 10, bottom: 0, right: 0)
-        return tableView
-    }()
 
     private let configuration: Configuration
 
     override var preferredStatusBarStyle: UIStatusBarStyle {
         return .default
     }
-    
+
     override var prefersStatusBarHidden: Bool {
         return false
     }
 
+    // MARK: - Subviews
+
+    private(set) lazy var searchBar: UISearchBar = {
+        let view = UISearchBar()
+        view.searchTextField.returnKeyType = .done
```

Comment:
It is better to use return key type `.search` for search 🤔

Reply:
Im use `.done` here because this button just hide keyboard and not trigger search 🤷🏼‍♂️ What do you think?

Reply:
Makes sense 👍